### PR TITLE
Build data generation and database seeding scripts

### PR DESCRIPTION
**Pull request by lhbrennan**

### CODE REVIEW

File: seeding/dataGeneration/dataGenHelpers.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const formatDate = function (date) {
```
Comment (jpizzo) [inline seeding/dataGeneration/dataGenHelpers.js]: Looks good.

File: seeding/dataGeneration/generateListingData.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  'price_per_night',
+  'star_rating',
+  'min_stay',
+  'cleaning_fee',
+  'max_guests',
+  'state',
+  'country',
+];
+
+fs.writeFileSync(dataFilePath, `${headers.join(',')}\n`, 'utf-8');
```
Comment (jpizzo) [inline seeding/dataGeneration/generateListingData.js:21]: I know you used sync and it works, but I would recommend going back and learning how to use the streams they are pretty handy.

File: server/index.js [hunk 3]
```diff
@@ -47,6 +47,33 @@ app.get('/api/rooms/:listingID/bookingInfo', async (req, res) => {
   }
 });
 
+/* ------------------------------------------------------------
+New API Routes
+-------------------------------------------------------------*/
+app.post('/api/rooms/:listingID/:bookingDate', (req, res) => {
+  const mySql = getConnection();
```
Comment (jpizzo) [inline server/index.js:54]: were these connections related to postgres? May rename them if they are to avoid confusion.

**Review by jpizzo — APPROVED** [review]

File: seeding/dataGeneration/generateListingData.js (hunk 2 above)
Comment (lhbrennan) [reply seeding/dataGeneration/generateListingData.js:21]: what would be the benefit in this case?

File: server/index.js (hunk 3 above)
Comment (lhbrennan) [reply server/index.js:54]: these were just the new CRUD connections that I created as the first step in SDC. not entirely sure they'll all even be hooked up.

**Pull request merged**